src/magellan_core/launch/passive.launch

```diff
@@ -7,8 +7,10 @@
     <node pkg="magellan_core" type="viz_turning_radius.py" name="viz_turning_radius" />
 
     <include file="$(find magellan_core)/launch/localization.launch" />
-    <include file="$(find magellan_core)/launch/lidar.launch" />
+    <!--<include file="$(find magellan_core)/launch/lidar.launch" />-->
```

Comment:
Can we check this in with lidar uncommented?

Reply:
Uncommented